source/coding-the-legislation/35_periods.md

```diff
@@ -1,17 +1,12 @@
-# Periods and instants
+# Periods and Instants
 
-A period can be a month, a year, `n` successive months, `n` successive years or the eternity.
-The smallest unit for OpenFisca periods is the **month**. Therefore:
+A `Period` can be a day, a month, a year, `n` successive days, `n` successive months, `n` successive years or the eternity.  
+An `Instant` is a specific day, such as a cutoff date.
 
-- All periods are presumed to start on the first day of their first month.
-- A period cannot be smaller than a month.
+You will mainly use `Period` objects.
```

Comment:
Most users will interact rarely with Period instances directly, but use strings for convenience. If the intent is to say that one deals mostly with Period rather than Instant objects, I'd discuss this only in the next section about Python objects.